### DISCUSSION TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'defaultFontFamily')

### DISCUSSION THREAD

**Discussion by koa73** [chartjs/chartjs-plugin-annotation#465] [Q&A] [answered]
I'm try to use your plugin but get this error just after load chartjs-plugin-annotation.js. I tried some examples in the end I tried your sample code https://github.com/chartjs/chartjs-plugin-annotation/blob/master/old_samples/box.html but result was the same.  Sorry may be it's stupid question and I lost some info how to use it

chartjs-plugin-annotation.js:485 Uncaught TypeError: Cannot read properties of undefined (reading 'defaultFontFamily')
    at Object.6../annotation.js (chartjs-plugin-annotation.js:485)
    at s (chartjs-plugin-annotation.js:10)
    at e (chartjs-plugin-annotation.js:10)
    at chartjs-plugin-annotation.js:10

**Comment by kurkle** [comment]
What version of the plugins are you trying to use and with what version of Chart.js?

**Comment by koa73** [comment]
* Chart.js v3.5.1
* Plugin Version: 0.5.7

**Comment by kurkle** [comment, marked as answer]
You need to use annotation v1+ with chart.js v3+

**Comment by koa73** [comment]
Thx
